web/src/main/java/com/es/phoneshop/web/controller/AjaxCartController.java

```diff
@@ -1,9 +1,13 @@
 package com.es.phoneshop.web.controller;
 
+import com.es.core.cart.CartAddDto;
+import com.es.core.cart.CartItemDto;
 import com.es.core.cart.CartService;
+import com.fasterxml.jackson.databind.exc.InvalidFormatException;
+import jakarta.validation.Valid;
```

Comment:
Use Spring MVC validation as per document with tasks, instead of jakarta.
https://docs.spring.io/spring-framework/docs/4.3.9.RELEASE/spring-framework-reference/htmlsingle/#mvc-config-validation